### PR TITLE
Michael/proceeding with boilerplate setup

### PR DESCRIPTION
**Pull request by MadOgre**
add assets and samples from package
add favicon
add absolute importing
add sample data
cleanup index.html (Remove references to manifest)
bring in fonts and create references in global.scss - import into index.tsx
create layout components (Shelf, Stack, Spacer)
create typography components
create exportable colors
create hamburger button and logo area components

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,25 @@
-import { HelloHell } from "./components/HelloHell";
+import { FrontendMentorMobileText, NoFeedbackMobileText } from "components/common/typography";
+import { HelloHell } from "components/HelloHell";
+import { Shelf, Spacer } from "components/common/layout";
```
Comment (victoriakario) [inline src/App.tsx]:
```suggestion
import { Shelf, Spacer } from "components/common/layout";
import { FrontendMentorMobileText, NoFeedbackMobileText } from "components/common/typography";

import { HelloHell } from "components/HelloHell";
```

Layout should some before typography and any other component. Maybe the basic building components should have a visual division from the components.

File: src/colors.module.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$standard-blue: #3A4374;
+$white: #FFF;
+
+:export {
+  standard-blue: $standard-blue;
+  white: $white;
+}
```
Comment (victoriakario) [inline src/colors.module.scss]:
```suggestion
}

```
All files should end with new-line character

File: src/components/common/typography/typography.module.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.FeedbackBoardMobileText {
+  font-size: 13px;
+  font-family: Jost;
+  line-height: 19px;
+  font-weight: 500;
+  opacity: 0.75;
+}
+
```
Comment (victoriakario) [inline src/components/common/typography/typography.module.scss]:
```suggestion
```

TOOO many new-lines now.

**Review by victoriakario — APPROVED** [review]
I see a lot of files that may need alphabetization. Please check before committing.

File: src/App.tsx (hunk 1 above)
Comment (MadOgre) [reply src/App.tsx]: Makes sense

File: src/components/common/typography/typography.module.scss (hunk 3 above)
Comment (MadOgre) [reply src/components/common/typography/typography.module.scss]: oops

**Comment by MadOgre** [comment]
> I see a lot of files that may need alphabetization. Please check before committing.

Yep you are right. I neglected to do it for this test run but will fix

**Review by victoriakario — APPROVED** [review]
LGTM

**Pull request merged**